code/datums/mutations/touch.dm

```diff
@@ -338,6 +349,55 @@
 		else
 			to_chat(mendicant, span_notice("Your veins swell!"))
 
+
+/datum/action/cooldown/spell/touch/lay_on_hands/proc/determine_if_this_hurts_instead(mob/living/carbon/mendicant, mob/living/hurtguy)
+	var/hurt_this_guy = FALSE
+
+	if(HAS_TRAIT(mendicant, TRAIT_PACIFISM))
+		return FALSE //always return false if we're pacifist
+
+	if(hurtguy.mob_biotypes & MOB_UNDEAD && mendicant.mob_biotypes & MOB_UNDEAD)
+		return FALSE //always return false if we're both undead //undead solidarity
+
+	if(hurtguy.mob_biotypes & MOB_UNDEAD && !HAS_TRAIT(mendicant, TRAIT_EVIL)) //Is the mob undead and we're not evil? If so, hurt.
+		hurt_this_guy = TRUE
+
+	else if(HAS_TRAIT(hurtguy, TRAIT_EVIL) && !HAS_TRAIT(mendicant, TRAIT_EVIL)) //Is the guy evil and we're not evil? If so, hurt.
+		hurt_this_guy = TRUE
+
+	else if(!(hurtguy.mob_biotypes & MOB_UNDEAD) && HAS_TRAIT(hurtguy, TRAIT_EMPATH) && HAS_TRAIT(mendicant, TRAIT_EVIL)) //Is the guy not undead, they're an empath and we're evil? If so, hurt.
+		hurt_this_guy = TRUE
+
+	return hurt_this_guy
+
+///If our target was undead or evil, we blast them with a firey beam rather than healing them. For, you know, 'holy' reasons. When did genes become so morally uptight?
+
+/datum/action/cooldown/spell/touch/lay_on_hands/proc/by_gods_light_i_smite_you(mob/living/carbon/paladin, mob/living/evil_motherfucker, smite_multiplier)
+	var/our_smite_multiplier = smite_multiplier
+
+	if(paladin.mind?.holy_role >= HOLY_ROLE_PRIEST)
+		var/possible_deity = GLOB.deity
+		var/mob/living/carbon/human/human_paladin = paladin //wow so original, dude
+		if(ishuman(human_paladin))
+			human_paladin.force_say()
```

Comment:
force_say empty here

Reply:
Yeah so, thats oddly how force_say() works. The forced text is below it.

Reply:
brooooooooo